utils/profile-actions.js

```diff
@@ -21,15 +21,20 @@ export function fillEmail(email) {
 
 export function fillProfile(
   options = {
+    firstName: 'Fernando',
     lastName: 'Coelho',
     country: 'BRA',
   }
 ) {
-  const { lastName = 'Coelho', country = 'BRA' } = options
+  const {
+    firstName = 'Fernando',
+    lastName = 'Coelho',
```

Comment:
wdyt about changing this? maybe use `auto auto` as some other tests already do

Reply:
i don't know if auto is a special keyword (i only remember "denied"), but yeah it would be just so the tests remains "agnostic" haha just a minor thought I had when looking at the changes